source/iface/command-accessor.rst

```diff
@@ -6,281 +6,290 @@
 
 .. _buffer-accessor:
 
-******************
+==================
 ``sycl::accessor``
-******************
+==================
 
 ::
 
-   template <typename dataT, int dimensions, sycl::access::mode accessmode,
-             sycl::access::target accessTarget = sycl::access::target::global_buffer,
-             sycl::access::placeholder isPlaceholder = sycl::access::placeholder::false_t>
-   class accessor;
+  class accessor;
+
+The ``accessor`` class provides access to data in a ``buffer`` from within a
+SYCL kernel function or from within a host task. When used in a SYCL kernel
+function, it accesses the contents of the buffer via the device’s
+global memory. These two forms of the accessor are distinguished
+by the ``AccessTarget`` template parameter. Both forms support
+the following values for the ``AccessMode`` template parameter:
+``access_mode::read``, ``access_mode::write`` and
+``access_mode::read_write``.
+
+.. seealso:: SYCL Specification |SYCL_SPEC_BUFFER_ACCESSOR|
 
-Description
+.. _access-targets:
 
-.. rubric:: Template parameters
+==============
+Access targets
+==============
 
-=================  =======
-``dataT``          Type of buffer element
-``dimensions``     Number of buffer dimensions
-``accessmode``     See :ref:`access-mode`
-``accessTarget``   See :ref:`access-target`
-``isPlaceholder``  True if accessor is a placeholder
-=================  =======
+.. list-table::
+  :header-rows: 1
 
-.. rubric:: Member types
+  * - Accessor target
+    - Meaning
+  * - ``target::device``
+    - Access a buffer from a SYCL kernel function via device global memory.
+  * - ``target::host_task``
+    - Access a buffer from a host task.
 
-===================  =======
-``value_type``       Type of buffer element
-``reference``        Type of reference to buffer element
-``const_reference``  Type of ``const`` reference to buffer element
-===================  =======
+============
+Member types
+============
+
+``accessor_ptr``
+================
+
+::
+
+  template <access::decorated IsDecorated> accessor_ptr
+
+If ``(AccessTarget == target::device):
+multi_ptr<value_type, access::address_space::global_space, IsDecorated>.``
+
+The definition of this type is not specified when
+``(AccessTarget == target::host_task)``.
```

Comment:
Please move this section until after the constructors and after the "member functions" sections.  It's less important than those other sections and may be confusing if described first

Reply:
Removed with https://github.com/KhronosGroup/SYCL_Reference/pull/94/commits/0c470ac494e971cde631e7ae798de38f0b11bdac